modules/controller_pool/assets/kubeconfig_copy.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-/usr/bin/ssh -i $ssh_private_key_path -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@$controller "while true; do if ! type kubeadm > /dev/null; then sleep 20; else break; fi; done"
-sleep 520
```

Comment:
@keithmattix How would the removal of this sleep impact your deployments?
I've added another condition to the loop to ensure that the config file is present before exiting.

Reply:
Yeah this should work; it's better than the arbitrary sleep anyway, so it probably needed to happen